emperor/support_files/js/controller.js

```diff
@@ -412,10 +490,11 @@ define([
    *
    * Helper method to resize the plots.
    *
-   * @param {Array} [dvdict] Dictionary of DecompositionViews.
-   * @param {function} [viewConstructor] Constructor of the view controller.
+   * @param {Array} dvdict Dictionary of DecompositionViews.
```

Comment:
Can you change {Array} to the {ITEM[]} style? From the doc string possibly {DecompositionView[]}?